### DISCUSSION TITLE
Changing commission rate

### DISCUSSION THREAD

**Discussion by tongson** [casper-network/casper-node#4846] [Q&A] [answered]
As a validator, how do we change the rate? Nothing on the docs, asking here.

**Comment by melpadden** [comment, marked as answer]
Hello. 

The concept of "commission" doesn't really exist in Casper, at least under that name. I'm not sure at this point if you're new to Casper, or if we are just using two different words to talk about the same thing.  What I'll do is, post some links to content that describes the rewards schem on Casper and hopefully that gets us closer to what you are looking for. 

In terms of validator rewards, there are articles on the current docs here: [https://docs.casper.network/economics/](https://docs.casper.network/economics/). The rewards model is changing for Condor, and there are some documents around the Knowledge base such as this one on Fee Elimination: [https://github.com/casper-network/condor-info/blob/main/articles/005-fee-elimination.md](https://github.com/casper-network/condor-info/blob/main/articles/005-fee-elimination.md)

Happy to help with further clarification.
